astropy/io/votable/validator/html.py

```diff
@@ -109,9 +109,7 @@ def write_votlint_warning(w, line, xml_lines):
     )
     if match:
         w.write(
-            "Line {:d}: {}\n".format(
-                int(match.group("line")), xml_escape(match.group("rest"))
-            )
+            f"Line {int(match.group('line')):d}: {xml_escape(match.group('rest'))}\n"
```

Comment:
`:d` is superfluous if one is guaranteed an integer (as here).

Reply:
Done